app/src/main/java/org/mozilla/fenix/home/pocket/PocketCategoriesViewHolder.kt

```diff
@@ -61,34 +60,36 @@ class PocketCategoriesViewHolder(
         val wallpaperState = components.appStore
             .observeAsComposableState { state -> state.wallpaperState }.value ?: WallpaperState.default
 
-        var selectedBackgroundColor: Color? = null
-        var unselectedBackgroundColor: Color? = null
-        var selectedTextColor: Color? = null
-        var unselectedTextColor: Color? = null
+        var (selectedTextColor, unselectedTextColor, selectedBackgroundColor, unselectedBackgroundColor) =
+            PocketStoriesCategoryColors.buildColors()
         wallpaperState.composeRunIfWallpaperCardColorsAreAvailable { cardColorLight, cardColorDark ->
             if (isSystemInDarkTheme()) {
-                selectedBackgroundColor = cardColorDark
-                unselectedBackgroundColor = cardColorLight
                 selectedTextColor = FirefoxTheme.colors.textActionPrimary
                 unselectedTextColor = FirefoxTheme.colors.textActionSecondary
+                selectedBackgroundColor = cardColorDark
+                unselectedBackgroundColor = cardColorLight
```

Comment:
I am looking at this and thinking for `PocketStoriesCategoryColors` and `buildColors`, we should just switch up the parameter orders so that the background color are before the text color. Just so we can avoid this reordering these assignments although we probably didn't have to reorder these assignments regardless.

No strong feelings here - you could opt to take no actions as well. It's mostly odd to me that we reordered these assignments to follow the parameter ordering. The main consideration here would be that we're changing git blame history for these particular lines due to the refactoring, but it's not a big deal here since we are very active in this area of the codebase and it's not particularly scary code that we haven't worked with in awhile.

Reply:
I think that's fair